.github/workflows/build.yml

```diff
@@ -127,17 +107,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
```

Comment:
using master may break the action at any time. Does the repo have a tag that we can use instead?

Reply:
It doesn't, this action uses the tag as a shorthand to select the toolchain version.